swarm/state/dbstore.go

```diff
@@ -46,6 +56,17 @@ func NewDBStore(path string) (s *DBStore, err error) {
 	}, nil
 }
 
+// NewInmemoryStore returns a new instance of DBStore. To be used only in tests and simulations.
+func NewInmemoryStore() *DBStore {
+	db, err := leveldb.Open(storage.NewMemStorage(), nil)
+	if err != nil {
+		panic(err)
```

Comment:
This is not great, by I didn't want to update the signature for `NewInmemoryStore` as we use it in many places in tests.